### ISSUE TITLE
Issue with running any RubyMine scripts - RE Addressable Gem

### ISSUE THREAD

**Issue by luke-hill** [sporkmonger/addressable#250]
I get the following error upon trying to run any RubyMine scripts

```
cannot load such file -- addressable/uri (LoadError)
```

Know there have been issues before with Capybara. However upon hovering over the Gem which I've now added I get the following remark:

```
Gem 'addressable' is not available in SDK 'ruby-2.2.5-p319' less... (Ctrl+F1) 
This inspection warns about unavailable gems inside the current SDK. The IDE needs the gems to provide code completion, navigation and analysis. An appropriate quick-fix is available to install the required gems.
```
I've tried changing Ruby version from 2.2.4 to 2.2.5 to 2.3.1 none work

**Comment by luke-hill** [comment]
As an update to this. Reverting back to (And force uninstalling 2.5.0), 2.4.0 fixes the issue. I've put a comment in my Gemfile to ensure we don't accidentally upgrade (Addressable was a dependency of Capybara, but without any restrictions)

**Comment by sporkmonger** [comment]
Hmm, that's interesting. The major change that differentiates 2.4.0 and 2.5.0 is that 2.5.0 no longer support Ruby 1.9.x, but it seems like you're using a supported version? I don't really use RubyMine, and I'm not at all confident that I'll be able to reproduce this, but I'm happy to fix if you track down root cause.

**Comment by sporkmonger** [comment]
If you update your `Gemfile` to use 2.5.0 again, do you get any errors on `bundle install`?

**Comment by luke-hill** [comment]
Steps performed.
1) Altered Gemfile (`2.5.0`)
2) `bundle`
--ERROR, It's locked at 2.4.0
3) `bundle update addressable`
--WORKED--


TL;DR - I "think" it's working. I checked when I physically ran this, and it was **before** public_suffix `2.0.4`

Could it be that perhaps 2.0.3 was the issue which is now fixed? Perhaps, can you leave this issue open for say 1-2 weeks and I will close if I find it is working fine. Apologies for the issues

Luke

**Comment by luke-hill** [comment]
Closing Issue. Legacy issue that should have been closed a while ago. Apologies.
